README.md

```diff
@@ -26,16 +26,16 @@ See the wiki for detailed documentation.
 Ensure that docker is up to date and initialized.
 Spin up mysql locally:
 ```
-docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+./docker/run-mysql.sh
```

Comment:
It does create a mysql docker image for me, but I need to access it using host=localhost and port=3310 rather than the defaults: that should either be noted here or changed.